### ISSUE TITLE
Cannot import Kafka ACL

### ISSUE THREAD

**Issue by leonardoam** [Mongey/terraform-provider-kafka#120]
I have the following ACL:
```terraform
resource "kafka_acl" "dataeng_all_cluster" {
  resource_name       = "kafka-cluster"
  resource_type       = "Cluster"
  acl_principal       = "User:dataeng"
  acl_host            = "*"
  acl_operation       = "All"
  acl_permission_type = "Allow"
}
```

I tried the following steps:
`terraform apply -target=kafka_acl.dataeng_all_cluster`

And then received:
```
kafka_acl.dataeng_all_cluster: Creating...
kafka_acl.dataeng_all_cluster: Creation complete after 2s [id=User:dataeng|*|All|Allow|Cluster|kafka-cluster|Literal]
...
Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
```

Then I removed it from state but kept on Kafka:
`terraform state rm kafka_acl.dataeng_all_cluster`

And tried to import it:
`terraform import kafka_acl.dataeng_all_cluster "User:dataeng|*|All|Allow|Cluster|kafka-cluster|Literal"`

Receiving the following error:
```
Error: Cannot import non-existent remote object

While attempting to import an existing object to
kafka_acl.dataeng_all_cluster, the provider detected that no object exists
with the given id. Only pre-existing objects can be imported; check that the
id is correct and that it is associated with the provider's configured region
or endpoint, or use "terraform apply" to create a new remote object for this
resource.
```

I'm using provider version 0.2.5.

**Comment by petoju** [comment]
@leonardoam PR is in https://github.com/Mongey/terraform-provider-kafka/issues/151

**Comment by Mongey** [comment]
Should be closed with the `v0.3.0` release
